docs/reference/upgrade/close-ml.asciidoc

```diff
@@ -29,4 +29,14 @@ POST _ml/set_upgrade_mode?enabled=true
 This method does not persist the absolute latest model state, rather it uses the
 last model state that was automatically saved. By halting the tasks, you avoid
 incurring the cost of managing active jobs during the upgrade and it's quicker
-than stopping {dfeeds} and closing jobs. 
\ No newline at end of file
+than stopping {dfeeds} and closing jobs. 
+
+//////////////////////////
+
+[source,js]
+--------------------------------------------------
+POST /_ml/set_upgrade_mode?enabled=false
```

Comment:
I think we'd be better off clearing this with `// TEARDOWN` in this case. Another option would be to teach `ESRestTestCase` to clear it between tests, but I don't think that is needed here because we don't commonly set this in tests.

Reply:
Thanks @nik9000 I've made that change

Reply:
I think we need `POST /_ml/set_upgrade_mode?enabled=true` to still be a user visible snippet.  Only `POST /_ml/set_upgrade_mode?enabled=false` should be a secret `TEARDOWN` snippet.  (I guess this is also complicated because `close-ml.asciidoc` gets included in other docs files, and I'm not sure how the `TEARDOWN`s interact when there are inclusions.)

Reply:
@droberts195 You can preview the changes here: 
https://doc-review-63bf6.firebaseapp.com/es2/rolling-upgrades.html
The snippet is still visible; have I misunderstood your concern?

Reply:
Sorry, yes, the preview looks fine.